collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -1447,7 +1448,7 @@ public void onClick(View v) {
 
     private void releaseOdkView() {
         if (odkView != null) {
-            odkView.releaseWidgetResources();
+            eventBus.post(new MediaPlayerRxEvent(RELEASE));
```

Comment:
What was the issue with releasing players? The new approach looks like it would have identical behavior but through a generic broadcast rather than targeted method calls.